spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -524,6 +524,29 @@
                 end
               end
             end
+
+            context 'SP requests enhanced_ipp_required' do
+              let(:vtr) { ['Pe'].to_json }
+
+              before do
+                allow(IdentityConfig.store).to receive(:openid_connect_redirect).
+                  and_return('server_side')
+                allow(IdentityConfig.store).to receive(:use_vot_in_sp_requests).and_return(true)
+                IdentityLinker.new(user, service_provider).link_identity(ial: 1)
+                user.identities.last.update!(
+                  verified_attributes: %w[given_name family_name birthdate verified_at],
+                )
+                allow(controller).to receive(:pii_requested_but_locked?).and_return(false)
+              end
+
+              it 'redirects to the redirect_uri immediately when pii is unlocked if client-side redirect is disabled' do
```

Comment:
This seems like an odd test-case here since PII being locked isn't necessarily related to enhanced in-person. I am not actually sure that we need changes to the authorization controller since its behavior should not change. It should continue forward the VTR along and we will operate on it downstream when we decide if we need enhanced in-person proofing.